src/pages/signin/SignInPage.tsx

```diff
@@ -295,11 +287,13 @@ function SignInPageInner({credentials, account, isInModal, activeClients, prefer
         </View>
     );
 }
-SignInPageInner.propTypes = propTypes;
-SignInPageInner.defaultProps = defaultProps;
+
 SignInPageInner.displayName = 'SignInPage';
 
-function SignInPage(props) {
+type SignInPageProps = SignInPageInnerProps & {navigation?: Partial<typeof navigationRef>};
```

Comment:
```suggestion
type SignInPageProps = SignInPageInnerProps;
```

`navigation` prop is not being used

Reply:
Correct but that prop is not being used by SigninPage. We shouldn't even pass it

Reply:
I'm not sure why would the perf test but I think we should still remove that prop since it's not really used by the App. Instead add a `@ts-expect-error` comment in the perf test